app/views/help/ab_testing.html.erb

```diff
@@ -9,9 +9,9 @@
     title: "This is a test page"
   } %>
 
-  <p class="govuk-body">We sometimes test different versions of pages with different users to see which work better. This page shows us if the process is working.</p>
+  <p class="govuk-body"><%= t('ab_testing.explanation') %></p>
 
-  <p class="govuk-body">There are 2 versions of this page. You are viewing the <strong class="ab-example-group"><%= @requested_variant.variant?("A") ? 'A' : 'B' %></strong> version.</p>
+  <p class="govuk-body"><%= t('ab_testing.version') %> <strong class="ab-example-group"><%= @requested_variant.variant?("A") ? 'A' : 'B' %></strong> version.</p>
```

Comment:
Do the letters `A` and `B` need translating?  We may want to use non-Latin characters in some languages.